ocp4/profiles/e8.profile

```diff
@@ -0,0 +1,141 @@
+documentation_complete: true
+
+title: 'Australian Cyber Security Centre (ACSC) Essential Eight'
+
+description: |-
+    This profile contains configuration checks for Red Hat Enterprise Linux CoreOS
+    that align to the Australian Cyber Security Centre (ACSC) Essential Eight.
+
+    A copy of the Essential Eight in Linux Environments guide can be found at the
+    ACSC website:
+
+    https://www.cyber.gov.au/publications/essential-eight-in-linux-environments
+
+selections:
+
+  ### Remove obsolete packages
+  #- package_talk_removed
+  #- package_talk-server_removed
+  #- package_xinetd_removed
+  #- service_xinetd_disabled
+  #- package_ypbind_removed
+  #- package_telnet_removed
+  #- service_telnet_disabled
+  #- package_telnet-server_removed
+  #- package_rsh_removed
+  #- package_rsh-server_removed
+  #- service_zebra_disabled
+  #- package_quagga_removed
+  #- service_avahi-daemon_disabled
+  #- package_squid_removed
+  #- service_squid_disabled
+
+  ### Software update
+  #- ensure_redhat_gpgkey_installed
+  #- ensure_gpgcheck_never_disabled
+  #- ensure_gpgcheck_local_packages
+  #- ensure_gpgcheck_globally_activated
+  #- security_patches_up_to_date
+  #- dnf-automatic_security_updates_only
+
+  ### System security settings
+  - sysctl_kernel_randomize_va_space
+#  - sysctl_kernel_exec_shield
```

Comment:
This check is not available for OCP4 platform. Also I believe it is not possible to use sysctl to turn off exec shield in RHCOS and RHEL8 and it defaults to enabled

Reply:
you can do sysctl setup with OCP4. There is an operator for that.

Reply:
My understanding is Red Hat has disabled the ability to turn exec shield off from RHEL7 onwards, so there is no need to check it is turned on: https://access.redhat.com/solutions/974563

Reply:
what i meant by 

> This check is not available for OCP4 platform.

is that ocp4 is not included as a platform for this rule:
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/rule.yml